modules/images-other-jenkins-agent-images.adoc

```diff
@@ -0,0 +1,37 @@
+// Module included in the following assemblies:
+//
+// * images/using_images/images-other-jenkins-agent.adoc
+
+[id="images-other-jenkins-agent-images_{context}"]
+= Jenkins agent images
+
+The {product-title} Jenkins agent images come in two varieties, Red Hat
+Enterprise Linux (RHEL) 7 based images and CentOS 7 Based Images.
+
+[discrete]
+=== RHEL 7 Based Images
+
+RHEL 7 Jenkins images are available through the Red Hat Registry:
+
+----
+$ docker pull quay.io/openshift3/jenkins-agent-maven-35-rhel7
+$ docker pull quay.io/openshift3/jenkins-agent-nodejs-8-rhel7
```

Comment:
No these images in quay.io, the jenkins images for 4.x are:
In the registry.redhat.io regsitry:
regsitry.redhat.io/openshift4/ose-jenkins:<v4.1.4>
regsitry.redhat.io/openshift4/ose-jenkins-agent-nodejs:<v4.1.4>
regsitry.redhat.io/openshift4/ose-jenkins-agent-maven:<v4.1.4>
regsitry.redhat.io/openshift4/ose-jenkins-agent-base:<v4.1.4>
In quay.io, the jenkins images  are stored with digest numbers like this:
$oc adm release info --pullspecs quay.io/openshift-release-dev/ocp-release:4.1.4  | grep  jenkins 
  jenkins                                       quay.io/openshift-release-dev/ocp-v4.0-art-dev@sha256:bec7f571b6ad293e697206786deb8194cd67d6bedf8e7eddb24290838807272a
  jenkins-agent-maven                           quay.io/openshift-release-dev/ocp-v4.0-art-dev@sha256:20e18317a9284549f56814d720da33393946f8f6add15f56974015d7abb78ba1
  jenkins-agent-nodejs                          quay.io/openshift-release-dev/ocp-v4.0-art-dev@sha256:98f342ba0aa7f6d012bf3f03d4a182aa92a1563ded39671b8c13f825e1348978